build.gradle

```diff
@@ -269,14 +269,28 @@ task distribute {
     dependsOn uploadUpdateVersion
 }
 
-task bintrayAar(type: GradleBuild) {
+// This is a work-around for a bug in the publish plugin that prevents publishing the same artifact twice in two
+// different publications
+task duplicateSourcesAndJavadoc(type: Copy) {
+    description = 'Prepare source and javadoc jars for publishing.'
+    group = 'Publishing'
+    dependsOn sourcesJar
+    dependsOn javadoc
+    from("${rootDir}/realm/realm-library/build/libs/") {
+        include '*.jar'
+        rename 'realm-android-library-(.*?)', 'realm-android-$1'
+    }
+    into "${rootDir}/realm/realm-library/build/libs/"
+}
+
+task bintrayAar(dependsOn: duplicateSourcesAndJavadoc, type: GradleBuild) {
```

Comment:
I actually added it as a method, but Gradle kept complaining it `could not find the property duplicateSourcesAndJavaDoc`. I am not entirely sure why, but doing it this way instead worked